arcgis-ios-sdk-samples/Edit data/Edit and sync features/EditAndSyncFeaturesViewController.swift

```diff
@@ -41,7 +37,6 @@ class EditAndSyncFeaturesViewController: UIViewController {
         }
     }
     
-    @IBOutlet private var toolBar: UIToolbar!
```

Comment:
This property isn't used in the code, so I removed it from both storyboard and source code.